plugin-bazel/src/main/kotlin/org/jetbrains/bazel/symbols/BazelTargetSymbolProvider.kt

```
@@ -0,0 +1,190 @@
+      }
+      
+    } catch (e: Exception) {
+      // Log and continue - don't fail indexing for individual files
+      // TODO: Add proper logging
+    }
+    
+    return symbols
+  }
+
+  private fun isBazelFile(indexableFile: IndexableFile): Boolean {
+    val fileName = indexableFile.fileName
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)